src/Avalonia.Native/WindowImplBase.cs

```diff
@@ -581,6 +582,11 @@ public virtual object TryGetFeature(Type featureType)
                 return AvaloniaLocator.Current.GetRequiredService<IClipboard>();
             }
 
+            if (featureType == typeof(ILauncher))
+            {
+                return new BclLauncher();
```

Comment:
Could the Launcher instance be cached?

Reply:
I only cached these launcher implementations that actually have any state in them (i.e. reference to the top level on Android). But I think I will change other implementations to singletones.